cpp/tests/CMakeLists.txt

```diff
@@ -19,7 +21,7 @@ endfunction(ConfigureTest)
 
 include_directories("${GTEST_INCLUDE_DIR}"
                     "${ARROW_INCLUDE_DIR}"
-                    "${FLATBUFFERS_INCLUDE_DIR}"
+                    "${FLATBUFFERS_ROOT}"
```

Comment:
Root directories should not be `include_directories`.

They should resemble targets as much as possible.

Reply:
All right!

Reply:
`${FLATBUFFERS_ROOT}` should be `${FLATBUFFERS_INCLUDE_DIR}` to match what is present stylistically.